README.md

```diff
@@ -0,0 +1,16 @@
+# Accelerator
+Accelerator replaces the default SRCDS crash handler with one that is a lot more reliable and uploads the crash reports to a community-accessible processing backend.
+This is useful since Valve's processing system discards crash reports from modded servers - and they still receive a representative sample of crash reports for actual game issues.
+
+The processing backend ([Throttle](https://crash.limetech.org/)) analyses crash reports to extract useful information, and in the case of common issues affecting many servers where manual analysis has been done, tags them with helpful notices with information on resolving the cause of the crash.
+
+## Installation
+- Click the Download link below.
+- Download the latest version for your server's OS.
+- Extract the Zip file and merge the addons/sourcemod directory with the one on your server.
+
+[Download](https://builds.limetech.org/?p=accelerator)
+
```

Comment:
I know that it's probably mentioned above but don't forget to include a section for when your server crashes what the server operators steps are to determine the cause. Actually probably better to just have the section show an example page of a crash and what kind of link you are to expect to see.